shared/build.gradle.kts

```diff
@@ -67,6 +100,42 @@ android {
     }
 }
 
-dependencies {
-    testImplementation(Dependency.JUNIT)
+val buildKonfigProperties = rootDir.loadGradleProperties("buildKonfig.properties")
+
+buildkonfig {
+    packageName = "co.nimblehq.ic.kmm.suv"
+
+    defaultConfigs {
+        buildConfigField(
+            STRING,
+            "CLIENT_ID",
+            buildKonfigProperties.getProperty("CLIENT_ID")
+        )
+        buildConfigField(
+            STRING,
+            "CLIENT_SECRET",
+            buildKonfigProperties.getProperty("CLIENT_SECRET")
+        )
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            buildKonfigProperties.getProperty("DEFAULT_BASE_URL")
+        )
+    }
+
+    defaultConfigs(Flavor.PRODUCTION) {
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            buildKonfigProperties.getProperty("PRODUCTION_BASE_URL")
+        )
+    }
+
+    defaultConfigs(Flavor.STAGING) {
+        buildConfigField(
+            STRING,
+            "BASE_URL",
+            buildKonfigProperties.getProperty("STAGING_BASE_URL")
+        )
+    }
```

Comment:
should we provide different client id vs secret for prod and staging?

Reply:
Good catch! I did not put this in mind, let's me update :+1:

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/86/commits/267561ece685f282ba111bf3132e937820df7d04